tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/util/WizardUtil.java

```diff
@@ -18,24 +18,46 @@
 import com.liferay.ide.core.templates.ITemplateOperation;
 import com.liferay.ide.core.templates.TemplatesCore;
 import com.liferay.ide.core.util.CoreUtil;
+import com.liferay.ide.core.util.FileUtil;
 import com.liferay.ide.project.core.ProjectCore;
 
 import java.io.ByteArrayInputStream;
 import java.io.InputStream;
 
 import org.eclipse.core.resources.IFile;
 import org.eclipse.core.resources.IFolder;
+import org.eclipse.core.resources.IProject;
 import org.eclipse.core.resources.IResource;
 import org.eclipse.core.runtime.CoreException;
 import org.eclipse.core.runtime.IProgressMonitor;
 import org.eclipse.wst.xml.core.internal.provisional.format.FormatProcessorXML;
 
 /**
  * @author Gregory Amerson
+ * @author Terry Jia
  */
 @SuppressWarnings("restriction")
 public class WizardUtil {
 
+	public static void addBundleInfo(IProject project) throws CoreException {
```

Comment:
hey  @jtydhr88 

This is specific to service builder wizard, it should not be added to a general wizard util class.  It should be isolated to the specific service builder operation.

Also this method name doesn't properly describe what this method is doing.  It is added the Liferay-Service header to the bnd.bnd file, so the method name should mention something to that affect.